src/languages/en.js

```diff
@@ -1,4 +1,174 @@
 export default {
     hello: 'Hello',
     phoneCountryCode: '1',
+    welcomeText: [
+        'With Expensify.cash, chat and payments are the same thing.',
+        'Money talks. And now that chat and payments are in one place, it\'s also easy.',
+        'Your payments get to you as fast as you can get your point across.',
+    ],
+    cameraPermissionRequired: 'Camera Permission Required',
+    noExtentionFoundForMimeType: 'No extension found for mime type',
+    problemGettingImageYouPasted: 'There was a problem getting the image you pasted',
+    updateApp: 'Update App',
+    // eslint-disable-next-line max-len
+    updatePrompt: 'A new version of Expensify.cash is available.\nUpdate now or restart the app at a later time to download the latest changes.',
+    cancel: 'Cancel',
+    upload: 'Upload',
+    yes: 'Yes',
+    no: 'No',
+    attachment: 'Attachment',
+    whoPaid: 'WHO PAID?',
+    whoWasThere: 'WHO WAS THERE?',
+    to: 'To',
+    whatsItFor: 'WHAT\'S IT FOR?',
+    optional: 'Optional',
+    split: 'Split',
+    request: 'Request',
+    nameEmailOrPhoneNumber: 'Name, email, or phone number',
+    new: 'NEW',
+    zoom: 'Zoom',
+    googleMeet: 'Google Meet',
+    search: 'Search',
+    uploadAttachment: 'Upload Attachment',
+    addAttachment: 'Add Attachment',
+    writeSomething: 'Write something...',
+    youAppearToBeOffline: 'You appear to be offline.',
+    copyToClipboard: 'Copy to Clipboard',
+    copied: 'Copied!',
+    copyLink: 'Copy Link',
+    markAsUnread: 'Mark as Unread',
+    editComment: 'Edit Comment',
+    deleteComment: 'Delete Comment',
+    beFirstPersonToComment: 'Be the first person to comment',
+    isTyping: ' is typing...',
+    and: ' and ',
+    areTyping: ' are typing...',
+    multipleUsers: 'Multiple users',
+    newChat: 'New Chat',
+    newGroup: 'New Group',
+    contacts: 'CONTACTS',
+    recents: 'RECENTS',
+    next: 'Next',
+    amount: 'Amount',
+    participants: 'Participants',
+    confirm: 'Confirm',
+    splitBill: 'Split Bill',
+    requestMoney: 'Request Money',
+    addYourPhoneToSettleViaVenmo: 'Add your phone number to settle up via Venmo.',
+    // eslint-disable-next-line max-len
```

Comment:
Pretty sure you can just disable this eslint rule for the whole file.
https://eslint.org/docs/user-guide/configuring/rules